optimum/intel/openvino/utils.py

```diff
@@ -485,13 +486,13 @@ def __init__(self, suffix=None, prefix=None, dir=None, ignore_cleanup_errors=Tru
         )
 
     @classmethod
-    def _cleanup(cls, name, warn_message, ignore_errors=False, delete=True):
+    def _cleanup(cls, name, warn_message, ignore_errors=True, delete=True):
```

Comment:
Why `ignore_errors=True` is needed? Is it safe to do so?

Reply:
this is needed because on windows time of running destructor for ov.Model happens after tmp dir cleanup by system if in python script there is no sys.exit(). Another possible solution is create tmp dir handle without automatic cleanup and remove it after we make sure that ov model is destroyed, in OVModelForXxx class destructor, but previously on my end such approach failed on some complex models (looks like something removed several times) with Segmentation Fault. Unfortunatly, I'm not replicated such case without optimum-intel yet to get advice from python API engineering team. It will be better to switch to this approach, but it requires more time for investigation, i think.